manta-pay/Cargo.toml

```diff
@@ -112,7 +110,7 @@ websocket = [
 [dependencies]
 aes-gcm = { version = "0.9.4", default-features = false, features = ["aes", "alloc"] }
 bip0039 = { version = "0.10.1", optional = true, default-features = false }
-bip32 = { version = "0.3.0", optional = true, default-features = false, features = ["bip39", "secp256k1"] }
+bip32 = { version = "0.4.0", optional = true, default-features = false, features = ["bip39", "secp256k1"] }
```

Comment:
Do you have any tests to confirm that this doesn't change the key values? I believe this caused issues before. Lets keep it at version 3 unless explicitly necessary for now.

Reply:
Apparently whatever was causing the incompatibility is no longer there.